templates/asc-alert-send-notification-email.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Is there a way to define my templates as "Preview"?
The connectors I use are "Preview" connectors

Reply:
Unfortunately no